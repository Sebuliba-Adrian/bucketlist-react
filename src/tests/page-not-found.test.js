import React from 'react';
import renderer from 'react-test-renderer';
import PageNotFound from '../components/page-not-found';

it('renders correctly', () => {
  const pageNotFound = renderer.create(<PageNotFound />).toJSON();
  expect(pageNotFound).toMatchSnapshot();
});
