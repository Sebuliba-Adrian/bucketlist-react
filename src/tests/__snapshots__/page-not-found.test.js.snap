// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="text-center mt-5"
>
  <h3
    className="text-cool-blue"
  >
    404 page not found
  </h3>
  <p
    className="text-muted"
  >
    We are sorry but the page you are looking for does not exist on bucketlists. Click 
    <a
      className="card-link text-cool-blue"
      href="/login"
    >
      here
    </a>
     to login.
  </p>
</div>
`;
